qiskit/providers/honeywell/honeywellprovider.py

```diff
@@ -104,7 +108,7 @@ def _discover_remote_backends(self):
             'backend_version': '0.0.1',
             'simulator': False,
             'local': False,
-            'basis_gates': ['rx', 'ry', 'rz', 'cx', 'h'],
+            'basis_gates': ['rx', 'ry', 'rz', 'cx', 'h', 'u1', 'x', 'y', 'u3'],
```

Comment:
What is the need for this change? Is this actually the basis set for the backends or are you trying to workaround other limitations?

Fwiw, the next version of terra which should be released shortly will include arbitrary basis translation to enable better support for basis other than `u1, u2, u3, cx` see Qiskit/qiskit-terra#4446 which should fix what I think the intent behind the change is here.

Reply:
This is trying to work around other limitations.  Without this qiskit was failing to submit at all.  When qiskit supports reduction to specific basis-gates we would be happy to revisit.